### PR TITLE
Testing Actor Systems doc updates

### PR DESCRIPTION
**Pull request by Danthar**
WIP for #2672

### CODE REVIEW

File: docs/articles/actors/testing-actor-systems.md [hunk 1]
```diff
@@ -1,134 +1,25 @@
----
-uid: testing-actor-systems
-title: Testing Actor Systems
```
Comment (alexvaluyskiy) [inline docs/articles/actors/testing-actor-systems.md]: Why have you removed the header?

File: docs/articles/actors/testing-actor-systems.md [hunk 2]
```diff
@@ -1,134 +1,25 @@
...
----
-
 # Testing Actor Systems
+
+
 As with any piece of software, automated tests are a very important part of the development cycle. The actor model presents a different view on how units of code are delimited and how they interact, which has an influence on how to perform tests.
 
-Akka.Net comes with a dedicated module `Akka.TestKit` for supporting tests at different levels, which fall into two clearly distinct categories:
```
Comment (alexvaluyskiy) [inline docs/articles/actors/testing-actor-systems.md]: Too many lines were removed, why?

File: docs/articles/actors/testing-actor-systems.md [hunk 3]
```diff
@@ -3,132 +3,29 @@ uid: testing-actor-systems
...
-[Fact]
-public void EndBackMessagesUnchanged()
-{
-    var echo = Sys.ActorOf(Props.Create<MyActor>());
-    echo.Tell("hello world");
-    ExpectMsg("hello world");
-}
-```
-The `TestKit` contains an actor named `MyActor` which is the entry point for messages to be examined with the various `ExpectMsg...` assertions detailed below. When mixing in the class `ImplicitSender` this test actor is implicitly used as sender reference when dispatching messages from the test procedure. The `MyActor` may also be passed to other actors as usual, usually subscribing it as notification listener. There is a whole set of examination methods, e.g. receiving all consecutive messages matching certain criteria, receiving a whole sequence of fixed messages or classes, receiving nothing for some time, etc.
+[!code-csharp[IntroSample](../../examples/DocsExamples/Testkit/TestKitSampleTest.cs?range=9-64)]
```
Comment (alexvaluyskiy) [inline docs/articles/actors/testing-actor-systems.md]: Don't use the ranges, use a naming region. Like
`[!code-csharp[WebStoreCustomerFSMActor.cs](../../examples/DocsExamples/Persistence/WebStoreCustomerFSMActor.cs?name=persistent-fsm-domain-events)]`

Comment (Danthar) [inline docs/articles/actors/testing-actor-systems.md]: ah naming region? How does that work ? using #region persistent-fsm-domain-events  #endregion ?

**Comment by Danthar** [comment]
This one is done for review. There are some minor TODO's left. But i'd rather leave those till after this is merged. As i want to see the end result on the docs page first.

**Pull request merged**